include/tvm/target/tag.h

```diff
@@ -131,6 +137,11 @@ inline TargetTagRegEntry& TargetTagRegEntry::set_config(Map<String, ObjectRef> c
   return *this;
 }
 
+inline TargetTagRegEntry& TargetTagRegEntry::add_config(String key, ObjectRef value) {
+  tag_->config.Set(key, value);
```

Comment:
Would you please also add the case when the key is already set?

Reply:
I think it is ok given the name is with_config. `Set` will work whether the key is already set.